.travis.yml

```diff
@@ -42,7 +39,7 @@ before_install:
     fi
   # update brew and pip on mac (note that mac is without nngt)
   - if [ "$TRAVIS_OS_NAME" = "osx" ]; then
-      brew install gcc@8 cmake python3 geos doxygen boost libomp;
+      brew install gcc cmake python3 geos doxygen boost@1.72 libomp;
```

Comment:
Actually it seems the @ is necessary; can you update it again to ``@10`` this time